packages/console/src/components/Topbar/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { version } from '../../../../core/package.json';
+
+export { version as currentVersion } from '../../../../core/package.json';
+
+/** Check if the target version is greater than the current Logto version. */
+export const isGreaterThanCurrentVersion = (target: string) => {
+  const latestComponents = (target.startsWith('v') ? target.slice(1) : target).split('.');
+  const currentComponents = version.split('.');
+
+  for (const [index, component] of latestComponents.entries()) {
```

Comment:
can we use `semver` to compare versions?

Reply:
thought about this. but it's yet another dependency which is kind of overkill currently.